Sources/UserDefaultsSerializable.swift

```diff
@@ -141,27 +142,52 @@ extension Set: UserDefaultsSerializable where Element: UserDefaultsSerializable
         self.map { $0.storedValue }
     }
 
-    public init(storedValue: [Element.StoredValue]) {
-        self = Set(storedValue.map { Element(storedValue: $0) })
+    public init?(storedValue: [Element.StoredValue]) {
+        self = Set(storedValue.compactMap { Element(storedValue: $0) })
     }
 }
 
 /// :nodoc:
 extension Dictionary: UserDefaultsSerializable where Key == String, Value: UserDefaultsSerializable {
     public var storedValue: [String: Value.StoredValue] {
-        self.mapValues { $0.storedValue }
+        self.compactMapValues { $0.storedValue }
     }
 
-    public init(storedValue: [String: Value.StoredValue]) {
-        self = storedValue.mapValues { Value(storedValue: $0) }
+    public init?(storedValue: [String: Value.StoredValue]) {
+        self = storedValue.compactMapValues { Value(storedValue: $0) }
     }
 }
 
 /// :nodoc:
 extension UserDefaultsSerializable where Self: RawRepresentable, Self.RawValue: UserDefaultsSerializable {
     public var storedValue: RawValue.StoredValue { self.rawValue.storedValue }
 
-    public init(storedValue: RawValue.StoredValue) {
-        self = Self(rawValue: Self.RawValue(storedValue: storedValue))!
```

Comment:
An added benefit of this change is that the `RawRepresentable` implementation no longer needs this force-unwrap